docs/documentation/_data/deckhouse-alerts.yml

```
@@ -4754,6 +4754,16 @@ alerts:
         At least one object violates the configured cluster security policies.
       severity: "7"
       markupFormat: markdown
+    - name: StandbyHolderDeploymentReplicasUnavailable
+      sourceFile: modules/040-node-manager/monitoring/prometheus-rules/standby-deployment-unavailable.yaml
+      moduleUrl: 040-node-manager
+      module: node-manager
+      edition: ce
+      description: Count of available replicas in Deployment standby-holder-{{$labels.node_group_name}} in namespace d8-cloud-instance-manager is at zero.
+      summary: |
+        Count of available replicas in Deployment standby-holder-{{$labels.node_group_name}} in namespace d8-cloud-instance-manager is at zero.
```

```suggestion
      description: Deckhouse has detected that there are no available replicas remaining in deployment standby-holder-{{$labels.node_group_name}} in namespace d8-cloud-instance-manager.
      summary: |
        No available replicas remaining in deployment standby-holder-{{$labels.node_group_name}} in namespace d8-cloud-instance-manager.
```
Let's change the wording a bit to make it consistent with the rest of extended-monitoring alerts.